src/test/java/SixPMTest.java

```diff
@@ -0,0 +1,102 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+
+public class SixPMTest {
+
+    private WebDriver driver = null;
+
+    @BeforeSuite
+    private void setSystemProperties() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver");
+    }
+
+    @BeforeMethod
+    private void createDriverInstance() {
+        driver = new ChromeDriver();
+    }
+
+    @AfterMethod
+    private void tearDownDriverInstance() {
+        if (driver != null) {
+            driver.quit();
+            driver = null;
+        }
+    }
+
+    @Test
+    private void test6PMMainFunctionalities() throws InterruptedException {
+        driver.get("https://www.6pm.com/");
+        driver.manage().window().maximize() ;
+
+        Actions actions = new Actions(driver);
+
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+
+        By accessoriesButtonLoc = By.xpath("//*[text()=\"Accessories\"]");
+        WebElement accessoriesButton = driver.findElement(accessoriesButtonLoc);
+        wait.until(ExpectedConditions.elementToBeClickable(accessoriesButtonLoc));
+        actions.moveToElement(accessoriesButton).perform();
+
+        By aviatorLoc = By.xpath("//*[text()=\"Aviators\"]");
```

Comment:
just write //*[text()='Aviators'] instead of this \"

Reply:
Addressed in 4a76f79.